app/helpers/template_helper.py

```diff
@@ -120,3 +121,13 @@ def safe_content(content):
         # Strip HTML Tags
         content = re.sub(r"</?[^>]+>", "", content)
     return content
+
+
+def get_data_layer(schema_theme):
+    if schema_theme == "census-nisra":
+        return "dataLayer = [{nisra : true}];"
+
+    if schema_theme == "census":
+        return "dataLayer = [{nisra : false}];"
+
+    return "dataLayer = [];"
```

Comment:
I think we should return a list instance and not a string, so:
```suggestion
    if schema_theme == "census-nisra":
        return [{"nisra": True}]

    if schema_theme == "census":
        return [{"nisra": False}]

    return []
```
Then in `_base.html` you can do:
```jinja
	<script nonce="{{ csp_nonce() }}">
      dataLayer = {{ data_layer | tojson }}
    </script>
```

Reply:
But do we want nisra key as a string? Proposed output is `[{nisra : [true | false] }]`..

Reply:
It is still a valid object. `{"nisra": false}` and `{nisra: false}` are identical in JS.

Reply:
Done! thanks!